packages/core/src/lib/transition/create-transition-callback.ts

```diff
@@ -295,8 +295,7 @@ export function createTransitionCallback<TAnimationValue = number>(
     runEntrance(element);
 
     return () => {
-      const cloned = element.cloneNode(true) as HTMLElement;
-      runExitTransition(cloned);
+      runExitTransition(element);
```

Comment:
There is a question.

Does the timing that React detach the original DOM element coincide with the timing that ref cleanup function is called? The DOM element can be moved by `insertClone` to different place. In that case, React cannot find that element for detaching.